website/src/parsers/mdx/mdxast.js

```diff
@@ -0,0 +1,48 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from '@mdx-js/mdx/package.json';
+import mdAstToMdxAst from '@mdx-js/mdx/md-ast-to-mdx-ast'
+
+const ID = 'MDXAST';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: 'https://mdxjs.com',
+  locationProps: new Set(['position']),
+
+  loadParser(callback) {
+    require(['@mdx-js/mdx'], callback);
+  },
+
+  parse(mdx, code) {
+    let result;
+
+    mdx.sync(code, {
+      mdPlugins: [
+        mdAstToMdxAst,
+        () => tree => {
+          result = tree;
```

Comment:
Even though I make the `mdx.sync` call, I only use the MDXAST tree. This is where I'm setting it, following how mdx works under the hood.